src/ActionsServiceProvider.php

```diff
@@ -15,15 +19,88 @@ class ActionsServiceProvider extends ServiceProvider
      */
     public $bindings = [
         'actions' => Action::class,
-        Actionable::class => Action::class,
+        Action::class => Action::class,
```

Comment:
no more binding with the interface ? Curious about this. Is there any other reason why except now being able to have container resolve classes for us ?